CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 30.5.1
+
+* Update popular links for London Bridge ([PR #3366](https://github.com/alphagov/frontend/pull/3366))
```

Comment:
Wrong PR - should be the one in publiishing components 
```suggestion
* Update popular links for London Bridge ([PR #2952](https://github.com/alphagov/govuk_publishing_components/pull/2952))
```

Reply:
ARGH good spot